scrolling_bug_on_long_code_cell.ipynb

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
<p>test comment on a long code cell to test scrolling on initial load.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/ipynb_playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>